src/Layers/xrRenderPC_GL/rgl.h

```diff
@@ -191,7 +191,7 @@ class CRender : public D3DXRenderBase
     R_sync_point q_sync_point;
 
     bool m_bMakeAsyncSS;
-    bool m_bFirstFrameAfterReset; // Determines weather the frame is the first after resetting device.
+    bool m_bFirstFrameAfterReset{ false }; // Determines weather the frame is the first after resetting device.
```

Comment:
When the default value is `0` or `false` or something like this, please, use `{}`.
```suggestion
    bool m_bFirstFrameAfterReset{}; // Determines weather the frame is the first after resetting device.
```